voila/configuration.py

```diff
@@ -85,3 +85,8 @@ class VoilaConfiguration(traitlets.config.Configurable):
     show_tracebacks = Bool(False, config=True, help=(
         'Whether to send tracebacks to clients on exceptions.'
     ))
+
+    multi_kernel_manager_class = Type(
+        default_value='jupyter_server.services.kernels.kernelmanager.AsyncMappingKernelManager',
+        klass='jupyter_client.multikernelmanager.MultiKernelManager'
```

Comment:
Should there be a small `help` string similar to the other traits?

Reply:
Good question, I considered it an expert feature, and the name self-describing, do you have a suggestion that adds information?

Reply:
Maybe something short like this is enough:

```python
help="""The kernel manager class. This is useful to specify a different kernel manager,
        for example a kernel manager with support for pooling.
        """
```